gctrpc/README.md

```diff
@@ -27,6 +27,19 @@ be toggled on or off depending on the users preference.
 
 ## Installation
 
+### Linux / macOS
+
+GoCryptoTrader requires a local installation of the `buf` cli tool that tries to make Protobuf handling more easier and reliable,
+after [installation](https://docs.buf.build/installation) you'll need to run:
+
+$ buf beta mod update
```

Comment:
Please encapsulate these with:

```sh
buf beta mod update
```

and the below command for easy of copying

Reply:
something like: `buf beta mod update; buf generate` ?
only gotcha is that `mod update` will tipically only need to be run once and then you iterate mostly with `generate`

Reply:
Instead of "$ buf beta mod update" (with the quotes removed) you can use:

"`shell
 buf meta mod update
"` (again with the quotes removed)

So that it allows a person to copy the command easily with the clipboard button on the right as shown below

![image](https://user-images.githubusercontent.com/4685270/127074200-7668caf8-a238-4cd0-a8de-e9322ea8767d.png)